[2025-03-08T20:26:07.779+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [queued]>
[2025-03-08T20:26:07.787+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [queued]>
[2025-03-08T20:26:07.788+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-03-08T20:26:07.788+0000] {taskinstance.py:1284} INFO - Starting attempt 7 of 8
[2025-03-08T20:26:07.788+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-03-08T20:26:07.799+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): print_date> on 2025-03-08 19:51:24.118312+00:00
[2025-03-08T20:26:07.803+0000] {standard_task_runner.py:55} INFO - Started process 77 to run task
[2025-03-08T20:26:07.808+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'print_date', 'manual__2025-03-08T19:51:24.118312+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/test_dag_ai.py', '--cfg-path', '/tmp/tmpkca9c9by']
[2025-03-08T20:26:07.808+0000] {standard_task_runner.py:83} INFO - Job 70: Subtask print_date
[2025-03-08T20:26:07.827+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-08T20:26:07.868+0000] {task_command.py:389} INFO - Running <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [running]> on host d781a364bdf5
[2025-03-08T20:26:07.919+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2025-03-08T19:51:24.118312+00:00
AIRFLOW_CTX_TRY_NUMBER=7
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-08T19:51:24.118312+00:00
[2025-03-08T20:26:07.920+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-03-08T20:26:07.920+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date']
[2025-03-08T20:26:07.935+0000] {subprocess.py:86} INFO - Output:
[2025-03-08T20:26:07.975+0000] {subprocess.py:93} INFO - Sat Mar  8 20:26:07 UTC 2025
[2025-03-08T20:26:07.976+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-08T20:26:07.996+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=print_date, execution_date=20250308T195124, start_date=20250308T202607, end_date=20250308T202607
[2025-03-08T20:26:08.823+0000] {util.py:67} INFO - error_code=missing_required_parameter error_message="Missing required parameter: 'messages'." error_param=messages error_type=invalid_request_error message='OpenAI API error received' stream_error=False
[2025-03-08T20:26:08.824+0000] {test_dag_ai.py:28} ERROR - LLM analysis failed: Missing required parameter: 'messages'.
[2025-03-08T20:26:08.832+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:120: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2025-03-08T20:26:08.832+0000] {email.py:229} INFO - Email alerting: attempt 1
[2025-03-08T20:26:10.455+0000] {email.py:241} INFO - Sent an alert email to ['monishassan1997@gmail.com']
[2025-03-08T20:26:12.223+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-03-08T20:26:12.267+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
